src/components/Header/_components/LoggedInHeader.tsx

```diff
@@ -0,0 +1,22 @@
+import { NotificationIcon } from '@/libs/utils/Icon';
+import Image from 'next/image';
+import Link from 'next/link';
+import React from 'react';
+
+import tripzipFavicon from '/public/logo/tripzipFavicon.png';
+
+export default function LoggedInHeader() {
+  return (
+    <div className="flex items-center gap-16">
+      <div className="cursor-pointer">
+        <NotificationIcon />
+      </div>
+      <Link href="/myPage" className="flex items-center gap-6 border-l-2 pl-16">
+        <div className="h-40 w-40 overflow-hidden rounded-full border">
+          <Image src={tripzipFavicon} alt="trip.zip" className="object-cover" />
```

Comment:
import 하지 않고 바로 경로를 쓰는 건 어때요?

```suggestion
          <Image src="/logo/tripzipFavicon.png" alt="trip.zip" className="object-cover" />
```

Reply:
이런식으로 하면 장점이 뭔가요? 🤔 이미지 최적화가 더 잘 되나요? 궁금합니다!

Reply:
저는 사실 import 하기가 귀찮아서 이렇게 하는 거를 선호했는데, 보미님 말씀 듣고 궁금해서 한 번 테스트해봤어요!

```jsx
<Image src={tripzipFavicon} width={20} height={20} alt="img" />
<Image src="/logo/tripzipFavicon.png" width={20} height={20} alt="img" />
```

![image](https://github.com/user-attachments/assets/3505f0cc-2451-4d22-b769-fdba319f7694)

초록색 중에 위에가 첫 번째고, 밑에가 두 번째인데 두 번째가 로드하는게 더 두 배 정도 짧긴 하네요!

성능 탭 좀 더 살펴봤을 때는 경로를 사용했을 때만 webp 적용, 이미지 사이즈 최적화가 된 것을 보아
아마도? 경로를 썼을 때 최적화 대상이 되는 것 같습니다. (확실하지 않아요 ㅎ)

Reply:
와... 이 사람 뭐지? 최고세요 그냥.. 당장 바꾸도록 하겠읍니다.. 👍🏻👍🏻👍🏻🙇🏻